mikupad.html

```diff
@@ -1611,12 +1626,26 @@
 				value=${value}
 				onChange=${({ target }) => onValueChange(JSON.parse(target.value))}
 				...${props}>
-				${options.map(o => html`<option
+				${(options = typeof options === 'function' ? options() : options).map(o => html`<option

```

Comment:
Made this able to evaluate expressions. Didn't end up using it here, but I thought I might as well leave it in.